### PR TITLE
Remove basic modules

### PR DESCRIPTION
**Pull request by KirillBogdanets**

### CODE REVIEW

File: src/apiActionSteps.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ import { sendHttpRequest } from './apiHelpers';
  * @param key key of the remembered value
  */
 When(
-  'I send {text} request to {landing-url}{headers} and save response as {text}',
```
Comment (AlexGalichenko) [inline src/apiActionSteps.ts:20]: I would suggest to wrap all values that were {text} previously to wrap into getValue function to extract memory value. That's give more flexibility in designing test cases.
and add such login into landingUrlType

File: src/apiVerificationSteps.ts [hunk 2]
```diff
@@ -9,7 +10,8 @@ import { expect } from 'chai';
  *
  * @param {String} statusCode should be valid status code
  */
-Then('Response {text} Status Code {validation} {text}', (response: any, validation: any, statusCode: string) => {
+Then('Response {response} Status Code {validation} {string}', (response: any, validationType: string, statusCode: string) => {
```
Comment (AlexGalichenko) [inline src/apiVerificationSteps.ts:13]: could you please rename validation with smth with prefix, like apiValidation.

File: src/apiActionSteps.ts (hunk 1 above)
Comment (KirillBogdanets) [reply src/apiActionSteps.ts:20]: I thought about adding `getValue()` to the steps where `{text}` type was used but I decided to ignore it due to it's not needed here. You will not use value from the memory for http methods for example

Comment (KirillBogdanets) [reply src/apiActionSteps.ts:20]: btw in `landingUrl` we have logic to get value from the memory if needed =)

**Review by AlexGalichenko — APPROVED** [review]

**Pull request merged**